package.json

```diff
@@ -73,7 +73,7 @@
   "peerDependencies": {
     "react": "^16.8.6 || ^17.0.0",
     "react-dom": "^16.8.6 || ^17.0.0",
-    "react-intl": "^5.25.1"
+    "react-intl": "^5.25.1 || ^6.4"
```

Comment:
nit: do we expect `6.4.4` to be the minimum (per the PR's title), or is any `6.4.x` OK? might suggest just adding `^6.4.0` if we allow any `6.4.x` so its consistent with the other peer dependencies' version granularity.

Reply:
6.4.4 is not required. any 6.4 will suffice. I agree with your suggestion to revise it to ^6.4.0.